src/components/Timeline/Event/Events.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
```

Comment:
I'm not sure why GitHub isn't picking up that this was a file moved from the `Timeline` folder. This is the evolution of `src/components/Timeline/Events.js`, jäh?

Reply:
Yes

Reply:
The way git tracks renames is by looking at file content. If the file was both renamed and substantially modified, then it has no way of knowing the previous one was an ancestor.

Reply:
I will be more careful about this in future

Reply:
@strand, if more than 50% of a file's content has been modified, then git won't track the rename. The use of `git mv` won't make a difference. The recommended approach is to do the rename in one commit, and the modification in another one.

If you are still curious, this is an easy thing to test:

```
cd Sia-EventUI
git mv README.md README2.md
git status
   renamed:    README.md -> README2.md
```

Now open the file and make some changes. Then:

```
git add .
git status
```

If the file was slightly modified it should still say:

```
   renamed:    README.md -> README2.md
```

If the file was substantially modified, it should say:

```
   deleted:    README.md
   new file:   README2.md
```

The reason this happens is because git is a content tracker, not a file tracker, so it doesn't really care about file or directory names.

The default threshold is 50%, but it is possible to list renames based on a different threshold, for example 30%:
```
git log -M30 -p --stat
```